docs/topics/api/stats.rst

```diff
@@ -0,0 +1,109 @@
+==========
+Statistics
+==========
+
+.. note:: This API requires :doc:`authentication <auth>`.
+
+The following API endpoints help you get your archived addon statistics.
+
+---------
+Archiving
+---------
+
+We are archiving statistics data that is older than one year.
+
+We currently archive the following data:
+
++---------------------+--------------------+---------------------------------------+
+| name                |     model_name     | description                           |
++=====================+====================+=======================================+
+| update counts       | updatecounts       | How many users updated this addon     |
++---------------------+--------------------+---------------------------------------+
+| download counts     | downloadcounts     | How many users downloaded this addon  |
++---------------------+--------------------+---------------------------------------+
+| theme update counts | themeupdatecount   | How many users downloaded this theme  |
++---------------------+--------------------+---------------------------------------+
+
+--------------------------
+List archived data monthly
+--------------------------
+
+The archive is structured by year/month, to see what data is archived
+for a specific month use the following api:
+
+.. http:get:: /api/v3/statistics/archive/[string:addon-slug]/[string:year]/[string:month]/
+
+    **Request:**
+
+    .. sourcecode:: bash
+
+        curl https://addons.mozilla.org/api/v3/statistics/archive/my-addon/2016/01/
+            -H 'Authorization: JWT <jwt-token>'
+
+    :param addon-slug: The slug for the add-on.
+    :param year: The year you want to fetch.
+    :param month: The month you want to fetch, please make sure to use 2 char months, e.g 01 instead of 1.
+
+    **Response:**
+
+    .. code-block:: json
+
+            [
+                {
+                    "date": "2016-01-18",
+                    "addon_id": 3615,
```

Comment:
Should it be the addon slug instead? It says "slug" below, but in the code I do see the id being returned.